.github/workflows/main.yml

```diff
@@ -0,0 +1,66 @@
+on:
+  push:
+    branches:
+      - master
+      - stable
+name: Build
+jobs:
+  build:
+    runs-on: macos-latest
+    name: Build and deploy
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/checkout@v2
+        with:
+          repository: AdaCore/gprbuild
+          path: gprbuild
+      - uses: actions/checkout@v2
+        with:
+          repository: AdaCore/gnatcoll-core
+          path: gnatcoll-core
+      - uses: actions/checkout@v2
+        with:
+          repository: AdaCore/gnatcoll-bindings
+          path: gnatcoll-bindings
+      - uses: actions/checkout@v2
+        with:
+          repository: AdaCore/langkit
+          path: langkit
+          fetch-depth: 0
+      - run: pushd langkit; git checkout ${{ github.ref }} || true; popd
+      - uses: ada-actions/toolchain@b55aaa7
```

Comment:
Why `@<specific sha1>` instead of `@dev` like written in Fabien's docs ?

Reply:
`@dev` doesn't include gnat community 2020. There is another branch `ce2020`, but it is unpublished yet on GH marketplace, so SHA1 hash protects us from stealing Bintray API key by Fabien :smiley:

Reply:
Ah, ok :) Thanks for the explanation